packages/kit-bg/src/services/ServiceAccount/ServiceAccount.ts

```
@@ -3969,35 +3972,227 @@ class ServiceAccount extends ServiceBase {
+                              ),
+                            );
+                          }
+                        }
+                      }
+                    }
+                  }),
+              );
+            }
+          }
+
+          if (keepWallet && keepWallet?.id && accountsToAddParams?.length) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)